content/cpp/concepts/strings/terms/replace/replace.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.replace()'
+Description: 'Returns a string with a portion replaced by another string.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `.replace()` function returns a string with a portion replaced by another string, or a portion of another string.
+
+## Syntax
+
+```pseudo
+basestring.replace(pos, len, str, subpos, sublen)
```

Comment:
Can we use camelcase for "basestring" in the syntax? I feel it's a little bit easier to read.
```suggestion
baseString.replace(pos, len, str, subpos, sublen)
```